<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemView</class>
 <widget class="QDialog" name="ItemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>791</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="homeButton">
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mListButton">
      <property name="text">
       <string>Museum list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="itemsViewedButton">
      <property name="text">
       <string>Items Viewed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="myCommentsButton">
      <property name="text">
       <string>My Comments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="accountButton">
      <property name="text">
       <string>Account</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="logoMM">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>0</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="muName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next Condensed</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;h1&gt;Museum Name&lt;/h1&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="iName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="iCreator">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Creator:</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User:</string>
   </property>
  </widget>
  <widget class="QLabel" name="comment">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>470</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Comment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back To Museum</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
